class-lifterlms.php

```diff
@@ -23,8 +23,13 @@
  * @since 4.0.0 Update session management.
  *              Remove deprecated class files and variables.
  *              Move includes (file loading) into the LLMS_Loader class.
+ * @since [version] Replace singleton code with `LLMS_Trait_Singleton`.
+ *
+ * @method static LifterLMS instance()
  */
-final class LifterLMS {
+final class LifterLMS  {
```

Comment:
(Remove extra space)

```suggestion
final class LifterLMS {
```